src/Tracking.php

```diff
@@ -566,4 +566,61 @@ private static function filter_add_to_cart_attributes( array $args, WC_Product $
 
 		return $args;
 	}
+
+
+	/**
+	 * Get the formatted warning message for the potential conflicting tags.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string The warning message.
+	 */
+	public static function get_third_party_tags_warning_message() {
+
+		$third_party_tags = self::get_third_party_installed_tags();
+
+		if ( empty( $third_party_tags ) ) {
+			return '';
+		}
+
+		return wp_kses_post(
+			sprintf(
+				/* Translators: %1$s Conflicting plugins */
+				__( 'The following installed plugin(s) can potentially cause problems with tracking: %1$s. %2$sRemove conflicting plugins%3$s or %4$smanage tracking settings%5$s.', 'pinterest-for-woocommerce' ),
+				implode( ', ', $third_party_tags ),
+				sprintf( '<a href="%s" target="_blank">', esc_url( admin_url( 'plugins.php' ) ) ),
+				'</a>',
+				sprintf( '<a href="%s" target="_blank">', esc_url( wc_admin_url( '&path=/pinterest/settings' ) ) ),
+				'</a>',
+			)
+		);
+
+	}
+
+
+	/**
+	 * Detect if there are other tags installed on the site.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return array The list of installed tags.
+	 */
+	public static function get_third_party_installed_tags() {
+
+		$third_party_tags = array();
+
+		if ( defined( 'GTM4WP_VERSION' ) ) {
+			$third_party_tags['gtm'] = 'Google Tag Manager';
+		}
+
+		if ( defined( 'PYS_PINTEREST_VERSION' ) ) {
+			$third_party_tags['pys'] = 'Pixel Your Site - Pinterest Addon';
+		}
+
+		if ( class_exists( '\Premmerce\WooCommercePinterest\PinterestPlugin' ) ) {
```

Comment:
We should add a `use` import statement at the top of the file, then use `PinterestPlugin::class` here instead of the string version of the namespace.

```suggestion
		if ( class_exists( PinterestPlugin::class ) ) {
```